config:
  pattern_file_options:
    num_segments: 2
    num_layers: 2
    num_bricks: 7
  construction_options:
    horizontal_spacing: 0.05
    vertical_spacing: 0.02  # Used only for GUI drawing
    brick_lengths:
      R: 0.3
      G: 0.6
      B: 1.2
      O: 1.8
    brick_height: 0.2
    brick_points:
      R: 1
      G: 2
      B: 3
      O: 4
    build_directions:  # 1: left-to-right, 0: automatically decide, -1: right-to-left
      segment_1: 0
      segment_2: 1
    max_pick_attempts: 2  # Maximum allowed consecutive fails for each brick color.
